examples/components-macos/bin/app.re

```diff
@@ -128,8 +126,9 @@ let () = {
     let window = Window.makeWithContentRect(0., 0., 680., 468.);
 
     let root = {
-      let view = BriskView.make();
       open Brisk.Layout;
+      let view = BriskEffectView.make();
```

Comment:
what if this is just a normal `BriskView`? Is the nested view still translucent?

Reply:
Yes.

Reply:
AMAZING!

Reply:
According to WWDC guy, nesting doesn't matter - effect views "punch through"!